README.md

```diff
@@ -6,10 +6,165 @@ developed as part of the DataWave project.
 The main working branch for this project is `datawave_ml` which was originally branched
 from the `cam6_3_139` tag.
 
+
+## Using this model
+
+### Obtaining CAM
+
+Clone a copy of this repository from git and ensure you checkout the `datawave_ml`
+branch on which this work is based:
+```
+git clone https://github.com/DataWaveProject/CAM.git
+cd CAM
+git checkout datawave_ml
+```
+This branch is built upon the `cam6_3_139` tag from the
+[main ESMCOMP/CAM repository](https://github.com/ESCOMP/CAM).
+
+### Preparing FTorch and setting the CIME component in CESM
```

Comment:
Maybe `CESM` needs to b replaced with `CAM`?

Reply:
I will address this in #6